src/jqLite.js

```diff
@@ -90,6 +90,16 @@
  * - [`val()`](http://api.jquery.com/val/)
  * - [`wrap()`](http://api.jquery.com/wrap/)
  *
+ * jqLite also provides a method restoring pre-1.8 insecure treatment of XHTML-like tags
+ * that makes input like `<div /><span />` turned to `<div></div><span></span>` instead of
```

Comment:
```suggestion
 * jqLite also provides a method, for restoring the pre-1.8 insecure treatment of XHTML-like tags,
 * that turns input like `<div /><span />` into `<div></div><span></span>` instead of
```

Reply:
Are the new commas valid? It looks a bit weird to me as the text separated by them isn't a side note but an important part of the sentence, defining the behavior.

Reply:
I don't see them as making that part of the sentence as a side note or something less important, but it's up to you. It's a long sentence and the commas give it some additional structure.

Reply:
I'm not an English native and I'm sometimes basing my writing on unspoken rules that may apply to my mother tongue - Polish - but not necessarily to English. That said, I had a look at:
https://research.ewu.edu/c.php?g=425370&p=2905512
and:
https://www.dailywritingtips.com/punctuation-mistakes-1-unnecessary-commas/
and, as I understand it, when a part of the sentence is essential, it shouldn't be separated by commas from both sides.

Maybe splitting this sentence into two would work better>

Reply:
Sounds good to me.